### PR TITLE
build scripts to download the models

### PR DESCRIPTION
**Pull request by gregorscholz**
Feel free to close this pr if it does not match your expectations.

Here i created some shell scripts you can execute before building an example to automatically download the models and, if necessary unzip them. 

### Usage
- execute `./scripts/download_model.sh` from the root of the example application.

### CODE REVIEW

**Comment by gregorscholz** [comment]
About the pipeline, maybe we could just execute the scripts in there like this:
```yml
- name: Downloading models
        run: |
          directories=$(find example -maxdepth 1 -type d)
          for dir in $directories; do
            cd $dir
            ./scripts/download_model.sh      
            cd -
          done
```

But feel free to suggest a different option!

File: example/object_detection_ssd_mobilenet/assets/model/lite-labelmap.txt [hunk 1]
```diff
@@ -1,90 +0,0 @@
-person
```
Comment (PaulTR) [inline example/object_detection_ssd_mobilenet/assets/model/lite-labelmap.txt:1]: Nice. So I know the image classification model has metadata on it to include the labels - did it work with this example?

File: example/object_detection_ssd_mobilenet/scripts/download_model.sh [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Delete old models
+rm -r assets/models/
+
+# Create folder if not already exists
+mkdir assets/models
```
Comment (PaulTR) [inline example/object_detection_ssd_mobilenet/scripts/download_model.sh:5]: So not having assets/models initially doesn't actually work with our CI. Maybe we can create the directory up front and put a README in each so it exist in the repo, then still have the scripts still for downloading the models.

File: example/object_detection_ssd_mobilenet/assets/model/lite-labelmap.txt (hunk 1 above)
Comment (gregorscholz) [reply example/object_detection_ssd_mobilenet/assets/model/lite-labelmap.txt:1]: Yes!

image

File: example/object_detection_ssd_mobilenet/scripts/download_model.sh (hunk 2 above)
Comment (gregorscholz) [reply example/object_detection_ssd_mobilenet/scripts/download_model.sh:5]: But what should be ignored then? We of course put `.tflite` in the .gitignore but should we also add `.txt`? 
Also, in the text_classification example, when we unzip the model it just returns `vocab` as a file without ending. 
Should i also just add this to `example/.gitignore`?

Comment (PaulTR) [reply example/object_detection_ssd_mobilenet/scripts/download_model.sh]: Closing this out for now - I _think_ CI might have fixed itself from another run, so rerunning it now to verify

**Comment by PaulTR** [comment]
Actually just realized you did add those text files to keep the folders, so that makes more sense :) Looks good to me, this is a great addition. Thanks!

**Pull request merged**